venvs/make.py

```diff
@@ -108,6 +134,10 @@ def main(
 
     for link in links:
         filesystem.link(
-            source=virtualenv.binary(name=link),
-            to=link_dir.descendant(link),
+            source=virtualenv.binary(name=link), to=link_dir.descendant(link)
+        )
+
+    if installs and config:
```

Comment:
Looks like there's a test missing here, but not sure I get the behavior anyhow. You don't persist if there's nothing to install (and even if someone explicitly said to) -- why's that?